pkg/yurthub/proxy/remote/remote.go

```diff
@@ -169,18 +170,25 @@ func (rp *RemoteProxy) modifyResponse(resp *http.Response) error {
 					resp.ContentLength = int64(size)
 					resp.Header.Set("Content-Length", fmt.Sprint(size))
 				}
+
+				// after un-gzip in filter, the header content encoding should be removed.
```

Comment:
I think "gunzip" is more accurate than "un-gzip".

Reply:
done